### ISSUE TITLE
Project Find Results icon

### ISSUE THREAD

**Issue by JoshBones** [wyze/atom-seti-icons#22]
Hey there,

Love the icons but I've noticed that when we do a Project Find, The results tab doesn't have the magnifying glass icon that it usually has, instead it has the generic text icon.

Is there any way we could get a magnifying glass back?

**Comment by wyze** [comment]
Which theme are you using? I am using One Dark and see the magnifying glass.

**Comment by JoshBones** [comment]
I'm using One Dark as well, I've also just noticed that the names of longer files are overrunning the tab width.

![tab title overrun](http://i.imgur.com/80KPf33.png?1)

Maybe I've got another plugin causing issues, if you aren't seeing these with the same theme @wyze then no need to worry about it. I'll mess around a bit and see if I have a conflicting plugin and if I find an issue I'll let you know.
